### PR TITLE
Update examples to use new frontend auth mechanism

### PR DESCRIPTION
**Pull request by lava**

### CODE REVIEW

**Review by dit7ya — APPROVED** [review]
LGTM!

File: examples/localdev/docker-compose.yaml [hunk 1]
```diff
@@ -110,6 +115,9 @@ services:
       - PRIVATE_WEBAPP_KEY=${TENZIR_PLATFORM_INTERNAL_APP_API_KEY}
       - AUTH_SECRET=${TENZIR_PLATFORM_INTERNAL_AUTH_SECRET}
       - PUBLIC_DISABLE_DEMO_NODE_AND_TOUR=${TENZIR_PLATFORM_DISABLE_LOCAL_DEMO_NODES}
+      - PRIVATE_DRIZZLE_DATABASE_URL=postgres://${TENZIR_PLATFORM_POSTGRES_USER}:${TENZIR_PLATFORM_POSTGRES_PASSWORD}@${TENZIR_PLATFORM_POSTGRES_HOSTNAME}/${TENZIR_PLATFORM_POSTGRES_DB}
```
Comment (dominiklohmann) [inline examples/localdev/docker-compose.yaml:118]: These are not env.example file.

File: examples/onprem/docker-compose.yaml [hunk 2]
```diff
@@ -80,6 +80,9 @@ services:
       PRIVATE_WEBAPP_KEY: ${TENZIR_PLATFORM_INTERNAL_APP_API_KEY}
       AUTH_SECRET: ${TENZIR_PLATFORM_INTERNAL_AUTH_SECRET}
       PUBLIC_DISABLE_DEMO_NODE_AND_TOUR: true
+      PRIVATE_DRIZZLE_DATABASE_URL: postgres://${TENZIR_PLATFORM_POSTGRES_USER}:${TENZIR_PLATFORM_POSTGRES_PASSWORD}@${TENZIR_PLATFORM_POSTGRES_HOSTNAME}/${TENZIR_PLATFORM_POSTGRES_DB}
```
Comment (dominiklohmann) [inline examples/onprem/docker-compose.yaml:83]: Same here.

File: examples/tenzir-developers/docker-compose.yaml [hunk 3]
```diff
@@ -96,6 +99,11 @@ services:
       - '5432:5432'
     volumes:
       - postgres_data:/var/lib/postgresql/data
+    healthcheck:
+      test: [ 'CMD-SHELL', 'pg_isready -U postgres' ]
```
Comment (dominiklohmann) [inline examples/tenzir-developers/docker-compose.yaml:103]: We use the long-form for arrays everywhere else in our YAML files. Mind rewriting all three instances of this?
```suggestion
      test:
        - CMD-SHELL
        - pg_isready -U postgres
```

**Review by dominiklohmann — CHANGES_REQUESTED** [review]

File: examples/localdev/docker-compose.yaml (hunk 1 above)
Comment (lava) [reply examples/localdev/docker-compose.yaml:118]: No, we're building this completely from the already existing variables.

File: examples/tenzir-developers/docker-compose.yaml (hunk 3 above)
Comment (lava) [reply examples/tenzir-developers/docker-compose.yaml]: :man_shrugging:

**Pull request merged**